ha/core/event_analyzer/filter/filter.py

```diff
@@ -182,15 +184,20 @@ def filter_event(self, msg: str) -> bool:
         """
         try:
             K8S_alert_required = False
-            message = json.loads(msg)
-
-            event_namespace = message.get("namespace")
-            event_pod_name = message.get("pod_name")
-
-            namespace = Conf.get(const.HA_GLOBAL_INDEX, f"K8S:POD{_DELIM}namespace")
-            pod_name = Conf.get(const.HA_GLOBAL_INDEX, f"K8S:POD{_DELIM}pods")
-            if event_namespace == namespace and event_pod_name.startswith(pod_name):
+            message = json.dumps(ast.literal_eval(msg))
+            message = json.loads(message)
+
+            Log.info('Received alert from fault tolerant')
+            # event_namespace = message.get("namespace")
+            event_resource_type = message.get("_resource_type")
```

Comment:
Here the resource type is "pod" .  Required resource type will be k8s:pod  as per the config file.
How are these two matching ?

Reply:
I think we already discussed this. And you are fine with this. Right?

Reply:
yes